tree/dataframe/inc/ROOT/RDataSource.hxx

```diff
@@ -204,7 +204,11 @@ public:
    // clang-format on
    virtual void Finalise() {}
 
-   virtual std::string GetDataSourceType() = 0;
+   /// \brief Return a string representation of the datasource type.
+   /// The returned string will be used by ROOT::RDF::SaveGraph() to represent
+   /// the datasource in the visualization of the computation graph.
+   /// Concrete datasources can override the default implementation.
+   virtual std::string GetLabel() { return "custom datasource"; }
```

Comment:
It's a stupid suggestion, but I think it is more coherent will all the other labels in the graph.
```suggestion
   virtual std::string GetLabel() { return "Custom Datasource"; }
```